openstack/barbican/templates/barbican-nanny-deployment.yaml

```diff
@@ -34,7 +34,6 @@ spec:
         {{- include "utils.topology.pod_label" . | indent 8 }}
       annotations:
         chart-version: {{.Chart.Version}}
-        checksum/etc-configmap.conf: {{ include (print $.Template.BasePath "/etc-configmap.yaml") . | sha256sum }}
         checksum/etc-configmap.conf: {{ include (print $.Template.BasePath "/scripts-configmap.yaml") . | sha256sum }}
```

Comment:
Removing the first `checksum/etc-configmap.conf` is an error. The second `checksum/etc-configmap.conf` should have been renamed to `checksum/scripts-configmap.conf`.